src/Utility/ChatClearer.js

```diff
@@ -0,0 +1,64 @@
+const BotModule = require("../BotModule");
+
+const interval = 60 * 1000;
+const defaultRetention = 60 * 1000;
+
+module.exports = BotModule.extend({
+    config: null,
+    dependencies: {
+        config: "config",
+    },
+    initialize: function (dependencyGraph) {
+        this._super(dependencyGraph);
+
+        this.discordClient.once("ready", () => {
```

Comment:
Probably overthinking, but can this be called more than once?

I don't think so bc I think when we disconnect we die and restart as opposed to reconnect

Reply:
All utilities use this same mechanic, and none of those have had issues in the past, so I think it's fine